src/lib.rs

```diff
@@ -128,6 +128,7 @@ pub fn get_tool_config(
         use_bz2: args.common.use_bz2,
         render_only: args.render_only,
         skip_existing: args.skip_existing,
+        cont_on_fail: args.cont_on_fail,
```

Comment:
I would personally not specify this in the tool configuration but instead just pass a boolean to the `run_build_from_args` function.